.github/workflows/backport.yml

```diff
@@ -6,8 +6,13 @@ on:
         description: "The pull request # to backport"
         required: true
 
+permissions:  # added using https://github.com/step-security/secure-repo
```

Comment:
I didn't follow how this is related to https://github.com/step-security/secure-repo?

Reply:
I use https://app.stepsecurity.io/secureworkflow to help configure permissions and hash